htdocs/test.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
What's this file here? Why do we need this information in the codebase?

Reply:
oops that shouldn't be there.